mmv1/templates/terraform/examples/snapshot_settings_specific_locations.tf.tmpl

```diff
@@ -0,0 +1,10 @@
+resource "google_compute_snapshot_settings" "{{$.PrimaryResourceId}}" {
+    project   = "{{index $.TestEnvVars "project"}}"
```

Comment:
here & in the other tests, the test project will be set by default & doesn't need to be set explicitly.

Reply:
Project will need to be set if you decide you need a non-default project, though!